GitUI/CommandsDialogs/RevisionFileTreeControl.cs

```diff
@@ -63,6 +65,7 @@ public RevisionFileTreeControl()
             _revisionFileTreeController = new RevisionFileTreeController(() => Module.WorkingDir,
                                                                          new GitRevisionInfoProvider(() => Module),
                                                                          new FileAssociatedIconProvider());
+            _rememberFileContextMenuController = RememberFileContextMenuController.Default;
```

Comment:
Can it be assigned in the definition above?

Reply:
ok